diff --git a/node_modules/react-native-particle-connect/lib/module/provider/index.js b/node_modules/react-native-particle-connect/lib/module/provider/index.js
index 5c91b63..398f81f 100644
--- a/node_modules/react-native-particle-connect/lib/module/provider/index.js
+++ b/node_modules/react-native-particle-connect/lib/module/provider/index.js
@@ -9,7 +9,6 @@ class ParticleConnectProvider {
   constructor(options) {
     this.options = options;
     _defineProperty(this, "events", new EventEmitter());
-    console.log(this.options, particleConnect);
     this.events.setMaxListeners(100);
   }
   on(event, listener) {
diff --git a/node_modules/react-native-particle-connect/src/provider/index.ts b/node_modules/react-native-particle-connect/src/provider/index.ts
index bb8f6fd..ae48954 100644
--- a/node_modules/react-native-particle-connect/src/provider/index.ts
+++ b/node_modules/react-native-particle-connect/src/provider/index.ts
@@ -10,7 +10,6 @@ class ParticleConnectProvider {
   private events = new EventEmitter();
 
   constructor(private options: ParticleConnectOptions) {
-    console.log(this.options, particleConnect);
     this.events.setMaxListeners(100);
   }
 
