diff --git a/node_modules/react-native-particle-auth/lib/module/provider/index.js b/node_modules/react-native-particle-auth/lib/module/provider/index.js
index bcddbaf..18d3424 100644
--- a/node_modules/react-native-particle-auth/lib/module/provider/index.js
+++ b/node_modules/react-native-particle-auth/lib/module/provider/index.js
@@ -9,7 +9,6 @@ class ParticleProvider {
   constructor(options) {
     this.options = options;
     _defineProperty(this, "events", new EventEmitter());
-    console.log(this.options, particleAuth);
     this.events.setMaxListeners(100);
   }
   on(event, listener) {
diff --git a/node_modules/react-native-particle-auth/src/provider/index.ts b/node_modules/react-native-particle-auth/src/provider/index.ts
index 3f53861..ca54320 100644
--- a/node_modules/react-native-particle-auth/src/provider/index.ts
+++ b/node_modules/react-native-particle-auth/src/provider/index.ts
@@ -10,7 +10,6 @@ class ParticleProvider {
   private events = new EventEmitter();
 
   constructor(private options: ParticleOptions) {
-    console.log(this.options, particleAuth);
     this.events.setMaxListeners(100);
   }
 
